### ISSUE TITLE
Current working directory content included in _moduleraw

### ISSUE THREAD

**Issue by rcurr** [cea-hpc/modules#220]
I'm using the environment-modules package in Ubuntu bionic (ver 4.1.1-1). If I create a file in my current working directory and run "bash" I get:

```
touch foo\ \(foo\)
bash

bash: eval: line 43: syntax error near unexpected token `('
bash: eval: line 43: ` foo (foo) _mlshdbg='' ;;'
bash: export: _moduleraw: not a function
bash: export: module: not a function
module: command not found
```

In /usr/share/modules/init/bash it seems to call:

```
36 eval `${_mlre:-}/usr/bin/tclsh /usr/lib/x86_64-linux-gnu/modulecmd.tcl bash autoinit`
```

running that command gives the "_moduleraw" function definition which has:

```
   if [ "${MODULES_SILENT_SHELL_DEBUG:-0}" = '1' ]; then
      case "$-" in
         *v*x*) set +vx; _mlshdbg='vx' ;;
         *v*) set +v; _mlshdbg='v' ;;
         *x*) set +x; _mlshdbg='x' ;;
         *) _mlshdbg='' ;;
      esac;
   fi;
```

and it looks like the "*"s need escaping. This seems to come from lines 3622-3628 of "/usr/lib/x86_64-linux-gnu/modulecmd.tcl" . Escaping them seems to fix things:

[modulecmd.tcl.patch.txt](https://github.com/cea-hpc/modules/files/2840855/modulecmd.tcl.patch.txt)

**Comment by rcurr** [comment]
Actually... quoting the "eval" in /usr/share/modules/init/bash  (line 36) seems to work as well:
```
eval `${_mlre:-}/usr/bin/tclsh /usr/lib/x86_64-linux-gnu/modulecmd.tcl bash autoinit`
```
 to

```
eval "$(${_mlre:-}/usr/bin/tclsh /usr/lib/x86_64-linux-gnu/modulecmd.tcl bash autoinit)"
```

in any case, happy to provide more info for you to reproduce this.

**Comment by xdelaruelle** [comment]
Thanks for this report. Everything was fixed regarding this issue in version 4.1.2. The issue was also reported through the Ubuntu bug system:

https://bugs.launchpad.net/ubuntu/+source/modules/+bug/1797345

As I said in the Ubuntu ticket, I encourage the package manager to update the Modules package on this LTS version of Ubuntu to the most recent v4.1 bugfix release (currently v4.1.4).

**Comment by rcurr** [comment]
@xdelaruelle thanks for that, I missed the ubuntu bug.
